### PR TITLE
Upgrade to Node 12

### PR DESCRIPTION
**Pull request by elias6**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@autofidev/ias",
-  "version": "0.1.2",
+  "version": "1.0.0-beta.0",
```
Comment (gairal) [inline package.json]: ⚡️

**Comment by paulgrock** [comment]
Do we know if this module is used anywhere?

File: test/client.spec.js [hunk 2]
```diff
@@ -15,13 +15,13 @@ describe('client.js', function() {
 		it('should fail with no loginName', function(done) {
 			assert.throws(function() {
 				new Client();
-			}, 'loginName is required');
+			});
```
Comment (gairal) [inline test/client.spec.js]: you might want to keep this test tho, just update

```js
assert.throws(() => (), { message: 'loginName is required' })
```

**Comment by elias6** [comment]
About whether this is used, I see it imported in a few places:
* https://github.com/AutoFi/loanapp/blob/master/lib/integrations/source.ias.js
* https://github.com/AutoFi/loanapp/blob/master/test/integrations.source.ias.spec.js
* https://github.com/AutoFi/loanapp/blob/master/test/task.products.spec.js

I also see it [mentioned in smart cow](https://github.com/AutoFi/smart_cow/blob/master/integration/tests/data/NMAC.json#L178) but it doesn't seem to be doing anything with it.

I am not sure whether any of these are necessary or important, but I can investigate if anyone wants.

**Comment by paulgrock** [comment]
> About whether this is used, I see it imported in a few places:
> 
> * https://github.com/AutoFi/loanapp/blob/master/lib/integrations/source.ias.js
> * https://github.com/AutoFi/loanapp/blob/master/test/integrations.source.ias.spec.js
> * https://github.com/AutoFi/loanapp/blob/master/test/task.products.spec.js
> 
> I also see it [mentioned in smart cow](https://github.com/AutoFi/smart_cow/blob/master/integration/tests/data/NMAC.json#L178) but it doesn't seem to be doing anything with it.
> 
> I am not sure whether any of these are necessary or important, but I can investigate if anyone wants.

It's probably not worth looking into unless upgrading to node 12 is a pain for this module. We should add a debt day ticket to the venom board to remove this from loanApp if not used though

**Review by gairal — APPROVED** [review]

File: test/client.spec.js [hunk 3]
```diff
@@ -15,13 +15,13 @@ describe('client.js', function() {
 		it('should fail with no loginName', function(done) {
 			assert.throws(function() {
 				new Client();
-			}, 'loginName is required');
+			}, { message: 'loginName is required' });
```
Comment (gairal) [inline test/client.spec.js]: not super important but use regexp preferably so it's compatible with 8 and 12

Comment (gairal) [reply test/client.spec.js]: I didn't know that when I put my first comment...

**Pull request merged**